src/dbus_python_client_gen/_invokers.py

```diff
@@ -352,15 +353,17 @@ def builder(namespace):
 
     return builder
 
-def make_class(name, spec):
+
+def make_class(name, spec, timeout=-1):
```

Comment:
This needs documentation, since "-1" means accept libdbus default timeout, rather than the more usual "keep going forever".